.gitignore

```diff
@@ -184,8 +184,6 @@ GitHub.sublime-settings
 
 bin/
 
-resources/orcid_app_info.json
```

Comment:
It may be safer to leave this until everyone has moved their resources directories, so things don't get committed accidentally.